src/Piece/Pawn.cpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "Piece/Pawn.h"
+
+bool Pawn::isValidMove(const Position src, const Position des) const {
+    if(isFirstMove)
```

Comment:
Simplify the conditions: Instead of having nested if-else blocks, you can simplify the conditions and return statements.